app/services/vot/supported_component_values.rb

```diff
@@ -38,6 +38,12 @@ module SupportedComponentValues
       implied_component_values: [P1],
       requirements: [:biometric_comparison],
     ).freeze
+    Pe = ComponentValue.new(
+      name: 'Pe',
+      description: 'Enhanced In Person Proofing is required',
+      implied_component_values: [P1],
```

Comment:
also what does P1 signify?

Reply:
I was understanding the "P" to signify proofing related components and "1" was because it was the initial component in the proofing category...I could be wrong though 😅

Reply:
hmm ok, i guess i wonder what `implied_component_values` is supposed to be? This file is totally new to me

Reply:
The description in this [PR](https://github.com/18F/identity-idp/pull/9971) is super helpful!

```
The P component represents identity proofing. It contains the following values:
1: Identity proofing is performed
```